Dockerfile

```
@@ -54,12 +56,15 @@ WORKDIR /usr/src/madara-orchestrator
 # Build the project
 RUN cargo build --release
 
+# Install Node.js dependencies for migrations
+RUN npm install
+
 
 FROM debian:bookworm
 
 # Install runtime dependencies
 RUN apt-get -y update && \
-    apt-get install -y openssl ca-certificates &&\
+    apt-get install -y openssl ca-certificates nodejs npm &&\
```

already installed above right?
```suggestion
    apt-get install -y openssl ca-certificates &&\
```

---

that was in the builder stage, you need to install it again here